### PR TITLE
Mint sign up page

### PR DESCRIPTION
**Pull request by NtwaliHeritier**
This is a clone of the mint sign up page found on www.mint.com.
We completed it using: 
     - HTML
     - CSS
     - Font Awesome Icons
We used flex box in our navigation bar to make it look good.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    <!--Sign up form-->
+    <section class="form">
+        
+        <h3>Create an Intuit account</h3>
+        <h5>One account for everything Intuit, including Mint.</h5>
+        <a href="#">Learn more </a>
+        <form>
+            <div> 
+                <label class="spacing" for="email">Email Address</label>
+                <input id="email" type="text">
```
Comment (MuriloRoque) [inline index.html]: This input should have the type 'email', not 'text'.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+                <label class="spacing" for="phone">Phone (Recommended)</label>
+                <input id="phone" type="tel">
+                <p class="sizing spacing">Standard call, messaging or data rates may apply</p>
+            </div>
+            <div class="last">
+                <label class="spacing" for="password">Password</label>
+                <input id="password" type="password">
+                <i class="fas fa-lock-open"></i>
+            </div>
+            <button class="nav-btn create">
```
Comment (MuriloRoque) [inline index.html]: This button should have the type 'submit'.

**Review by MuriloRoque — CHANGES_REQUESTED** [review]
# Status: Changes requested :red_circle: 

## Hello @NtwaliHeritier and @Nkuria :wave: 

## You have done an amazing job, your page looks awesome, I'm impressed :clap: :clap: :clap: 

## We only need to fix two MINOR issues:

- [ ] Please check my comments in your HTML file below, these are requirements ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/html_forms/form_and_input_requirement.png))

- [ ] Please, add a live link to your README file, try using [Raw Githack](https://raw.githack.com/)

## Apart from that, everything is great! :grin: 

## Kindly make the requested changes and submit for another review, please :pray: 

## I will be on Slack 24/7 in case you have any questions about this review :handshake: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Review by lmaldonadoch — CHANGES_REQUESTED** [review]
# Changes Required 🥇 🏃‍♂️

## Great job, team!!! 🏆

Your clone is almost identical to the original website; it is quite impressive! :clap: :clap:

There is a small detail that we need to take care of before being able to move on to the next project. Here it goes:

- [ ] [REQUIRED] The product row is not meant to be an image; it should be three pairs of images with the corresponding text.
image


- [ ] [OPTIONAL] There is a slightly different color at the top of the page. See if you can implement it!
image

- [ ] [OPTIONAL] The texts below the phone field, privacy and terms, and reCAPTCHA are larger than on the original website. Try to make it more alike!
image



You guys did an amazing job, keep it going! :clap:

Please implement the required changes, and think about the optional ones.

It was a pleasure reviewing your code,

Happy coding! 💻

Slack: Luis Angel Maldoando
[LinkedIn](https://www.linkedin.com/in/lmaldonadoch/)

**Review by MuriloRoque — APPROVED** [review]
# Status: Approved :green_circle: 

## Hello @NtwaliHeritier and @Nkuria :wave: 

## Thank you for making all of the requested changes, your project is approved! You may merge, good luck with your next projects :rocket: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Pull request merged**